opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipEditionOverview.tsx

```diff
@@ -250,8 +250,8 @@ const StixSightingRelationshipEditionOverviewComponent: FunctionComponent<Omit<S
     const inputValues = Object.entries({
       ...otherValues,
       confidence: parseInt(String(values.confidence), 10),
-      first_seen: formatDate(values.first_seen),
-      last_seen: formatDate(values.last_seen),
+      first_seen: values.first_seen ? formatDate(values.first_seen) : null,
```

Comment:
useless because formatDate already check if date is null

Reply:
Corrected